index.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <main id="main">
+      <h1 id="title">Dr. Norman Borlaug</h1>  <!-- preferrably an h1 tag than a p tag-->
```

Comment:
An h1 tag would be better than the p tag for semantic structure

Reply:
- [ ] Kindly formulate your comment clearer, giving also the reason why you require changes :recycle:

Reply:
Kindly make use of a h1 tag instead of a p This helps screen readers, search engines, and other user devices decide the context and significance of your website.